### PR TITLE
chore: updating dependabot file to support GHA, Python and docker packages

### PR DESCRIPTION
**Pull request by rajpalc7**

### CODE REVIEW

**Comment by rajpalc7** [comment]
Hi @WadeBarnes , this is ready for review now

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,20 +1,35 @@
...
+        update-types: ["version-update:semver-major"]
+
+  - package-ecosystem: "docker"
+    directory: "/aries-mobile-tests"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
+
```
Comment (WadeBarnes) [inline .github/dependabot.yml:35]: @rajpalc7, There is no `/aries-mobile-tests` folder in this project.  It looks like this code as direct copy of your work here; https://github.com/openwallet-foundation/mobile-wallet-test-harness/pull/253/files

**Review by WadeBarnes — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (rajpalc7) [reply .github/dependabot.yml]: @WadeBarnes Ready for review now.

**Review by WadeBarnes — APPROVED** [review]
Much better.  Thanks

**Pull request merged**